Makefile

```diff
@@ -12,6 +12,7 @@ setup-git:
 
 test:
 	SNUBA_SETTINGS=test pytest -vv tests -v -m "not ci_only"
+	cd snuba/admin && yarn install && yarn run test
```

Comment:
Let's not add this here. Most changes don't involve the frontend and I don't want to slow down normal development to run these tests.